StoryTowerWeb/schemas/mutations.js

```diff
@@ -29,6 +32,52 @@ const addScrapedDataToCatalog = async ({ scrapedData }) => {
   }
 };
 
+const createUser = async (_, { email, username, password }) => {
+  try {
+    const existingUser = await User.findOne({ $or: [{ email }, { username }] });
+    if (existingUser) {
+      throw new Error('User with this email or username already exists.');
```

Comment:
do we want to throw here? won't that crash? we could `console.error` or just return an error message to UI instead

Reply:
This just puts an error in the console. I didnt see any crashing.